packages/icons/package.json

```diff
@@ -4,13 +4,44 @@
   "main": "dist/TalendIcons.js",
   "types": "dist/typeUtils.d.ts",
   "style": "build/octicons.css",
+  "module": "./lib-esm/index.js",
+  "exports": {
+    "./package.json": "./package.json",
+    "./.storybook/main.js": "./.storybook/main.js",
+    "./.storybook/preview.js": "./.storybook/preview.js",
```

Comment:
I don't think these ones are needed. As long as the files are part of the npm package they should be importable by referring to the file names.

Reply:
Storybook try to resolve them reading the package.json and (if we remove those lines) as it's not define, there is an error: 
```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './.storybook/main.js' is not defined by "exports" in /Users/smouillour/Developments/Talend/TUI/ui.worktrees/jmfrancois/feat/add-esm-support-arch-737/node_modules/@talend/icons/package.json
```

but at least, it looks like the preview can be removed